iphone/Classes/defines.h

```diff
@@ -131,3 +131,35 @@
 #define USE_TI_UIREFRESHCONTROL
 #define USE_TI_UIIOSAPPLICATIONSHORTCUTS
 #define USE_TI_UIIOSBLURVIEW
+
+#define USE_TI_MEDIASHOWCAMERA
+#define USE_TI_MEDIAHIDECAMERA
+#define USE_TI_MEDIAOPENPHOTOGALLERY
+#define USE_TI_MEDIATAKEPICTURE
+#define USE_TI_MEDIASTARTVIDEOCAPTURE
+#define USE_TI_MEDIASTOPVIDEOCAPTURE
+#define USE_TI_MEDIASWITCHCAMERA
+#define USE_TI_MEDIAREQUESTCAMERAACCESS
+#define USE_TI_MEDIAREQUESTCAMERAPERMISSIONS
+#define USE_TI_MEDIAHASCAMERAPERMISSIONS
+#define USE_TI_MEDIAHASPHOTOGALLERYPERMISSIONS
+#define USE_TI_MEDIAREQUESTPHOTOGALLERYPERMISSIONS
+#define USE_TI_MEDIAOPENMUSICLIBRARY
+#define USE_TI_MEDIAHIDEMUSICLIBRARY
+#define USE_TI_MEDIAQUERYMUSICLIBRARY
+#define USE_TI_MEDIAREQUESTAUDIOPERMISSIONS
+#define USE_TI_MEDIAHASAUDIOPERMISSIONS
+#define USE_TI_MEDIAREQUESTAUTHORIZATION
+#define USE_TI_MEDIAHASMUSICLIBRARYPERMISSIONS
+#define USE_TI_MEDIAREQUESTMUSICLIBRARYPERMISSIONS
+#define USE_TI_MEDIACANRECORD
+#define USE_TI_MEDIAISCAMERASUPPORTED
+#define USE_TI_MEDIAISMEDIATYPESUPPORTED
+#define USE_TI_MEDIASAVETOGALLERY
+#define USE_TI_MEDIASTARTVIDEOEDITING
+#define USE_TI_MEDIAAUDIOSESSION
```

Comment:
Based on testing on device using KitchenSInk, This define is not necessary since it doesn't get generated when building to device, even when the app manages audio player.